pydata-chicago-2016/videos/a-year-of-pyxley-my-first-open-source-adventure.json

```diff
@@ -0,0 +1,23 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "PyData Chicago 2016\n\nLast year, I created my first Flask-powered web application utilizing React.js and D3. I refactored it into reusable components and released it as Pyxley. In this talk, I\u2019ll introduce the basics of Pyxley and discuss what has and hasn\u2019t worked. More importantly, I\u2019ll talk about what it\u2019s like to maintain an open-source project for the first time.",
+  "duration": 1629,
+  "language": "eng",
+  "recorded": "2016-08-28",
+  "related_urls": [
+    "http://pydata.org/chicago2016/schedule/presentation/26/"
+  ],
+  "speakers": [
+    "Nicholas Kridler"
+  ],
+  "tags": [
+    "adventure",
+    "Open Source"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/Y17wDH-IaG0/maxresdefault.jpg",
+  "title": "A Year of Pyxley: My First Open Source Adventure",
+  "videos": {
```

Comment:
Almost ok, but videos is an array, to pass the tests it should be (for all files) like this:
~~~ json
  "videos": [
    {
      "type": "youtube",
      "url": "https://www.youtube.com/watch?v=Y17wDH-IaG0"
    }
  ]
~~~